src/app/components/study-accordion/study-accordion.directive.js

```diff
@@ -64,4 +64,11 @@ angular.module('transmartBaseUi')
         }
       };
 
+      $scope.clearMetadata = function () {
+        $scope.metadataObj = {};
+        var query = document.getElementsByClassName("popover ng-isolate-scope right fade in");
```

Comment:
Is it necessary to select on all these classes? Wont 'popover' suffice? ng-isolate-scope is almost always present on directive generated elements.